### ISSUE TITLE
Provide markdown version of the LPPL license

### ISSUE THREAD

**Issue by slel** [latex3/latex2e#457]
Providing a markdown version of the LPPL license would help
display it nicely in code repositories that use it.

Current `LICENSE` file renamed to `LICENSE.md` displays poorly:

- [Markdown rendering of LPPL license file](https://github.com/dagstuhl-publishing/styles/blob/v2021.0.1/LICENSE.md)

**Comment by josephwright** [comment]
I've transferred to 2e as this is about the 'official' file

**Comment by igreil** [comment]
Any update on this? I'd really like to see this license in MarkDown format. I would've done it myself, but **modification of it is not allowed** scared me away …

**Comment by FrankMittelbach** [comment]
No, there is no update and there will not be any most likely. Markdown is simply not rich enough to easily capture the different internal links in the license so that depending on the markdown reader it delivers rubbish. I have fore have given up on trying to provide one.

I suggest to refer to the cannonical link for the license or include it as the official .txt file.

**Comment by igreil** [comment]
_Markdown is simply not rich enough to easily capture the different internal links in the license_

Not sure I understand what you mean: Markdown can do anything that plain text can, and then some. Even if there is no way to ensure automatic hyperlinks for all environments, textual references would continue to work exactly the same as in the .txt version. But thanks for clearing that up, I might have to reconsider other licenses.

**Comment by FrankMittelbach** [comment]
If I avoid using numbered lists then yes, I see that the glp used this approach of saying something like

- a) ...

but it gives a rather odd representation. However, not sure what markdown or not markdown has to do with choosing a license. LPPL is not asking you to include it with your work, linking to https://www.latex-project.org/lppl.txt gives your users direct access to the license.
